packages/agreement-process/src/services/validators.ts

```diff
@@ -0,0 +1,166 @@
+import {
```

Comment:
@galales my intent with this file is to collect all validation related to agreements, by the way, we can evaluate to ship this functions in `commons` module if necessary.
Main doubt persists, in [original scala code](https://github.com/pagopa/interop-be-agreement-process/blob/66781549a6db2470d8c407965b7561d1fe493107/src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala#L1060) there are nested invocation and overloaded between methods, in the future we can evaluate to simplify them.

Reply:
@galales because this topic isn't main PR focus, for now I'll proceed with merge; furthermore if there was any concern about this file or validation methods usages and composition, discuss them together.